DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyCheckBox.cs

```diff
@@ -101,7 +101,11 @@ public override void ParseAttributeFromINI(IniFile iniFile, string key, string v
                     UserChecked = checkedValue;
                     return;
                 case "DisallowedSideIndex":
-                    DisallowedSideIndex = Conversions.IntFromString(value, DisallowedSideIndex);
+                    DisallowedSideIndices = new int[]{ int.Parse(value) };
+                    return;
+                case "DisallowedSideIndices":
+                    string[] sides = value.Split(',');
+                    DisallowedSideIndices = Array.ConvertAll(value.Split(','), (s) => { return Conversions.IntFromString(s, -1); });
                     return;
```

Comment:
Actually with the current way it's implemented the tags used will conflict when both are used. I think appending should be done instead of replacement.

Reply:
Is there a reason to ever use both though?

Reply:
That's for backwards compatibility and to make it foolproof.